packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -38,9 +38,15 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         if (this.mouseIsDown && this.props.unitController.selectionRange) {
             const xStartPos = this.props.unitController.selectionRange.start;
             const scale = this.props.viewRange.getEnd() - this.props.viewRange.getstart();
+            let end = xStartPos + ((event.screenX - this.posPixelSelect) / this.lineChartRef.current.chartInstance.width) * scale;
+            if (end <0) {
```

Comment:
one more space formatting, please add a space after <.

Reply:
yes didn't see it thank you